open_instruct/grpo_vllm_thread_ray_gtrl.py

```diff
@@ -954,14 +987,13 @@ def generate_with_engines(prompts: List[List[int]], sampling_params: SamplingPar
         )
         stats_shape = (
             args.num_epochs,
-            args.num_mini_batches * args.number_samples_per_prompt,
+            args.num_mini_batches,
```

Comment:
This fixes 

![image](https://github.com/user-attachments/assets/a9bba328-e8b4-4aa1-9415-db1cd979b44c)


to 

![image](https://github.com/user-attachments/assets/9210418b-954c-45b2-8b9d-48260fa2f5c6)